### ISSUE TITLE
Show a meaningful error on missing element

### ISSUE THREAD

**Issue by gaearon** [kadirahq/react-storybook#532]
Can you spot a mistake here?

```js
storiesOf('App', module)
  .add('normal', () => {
    <App />
  });
```

I used `{` instead of `(`, and so didn’t return anything.
As a result, I got this error:

<img width="1118" alt="screen shot 2016-10-05 at 11 47 29" src="https://cloud.githubusercontent.com/assets/810438/19110383/82799b68-8af1-11e6-9ce8-6d962290d05b.png">

I think Storybook should validate elements before passing them to React. If element is undefined, it should throw something like "Expected the function in `storiesOf().add(...)` to return a React element. Instead it returned undefined. Did you forget the return statement?"

Similarly, if `element.type` is `undefined`, print something like "Expected the function in `storiesOf().add(...)` to return a valid React element. However, element's type is undefined. If you imported that component without the curly braces, verify that it was exported with an "export default" syntax."

**Comment by arunoda** [comment]
This is pretty great.
And I think many people would have met with this error.
I can do a fix.

**Comment by arunoda** [comment]
Fixed with `v2.22.0`
